code_search/kubeflow/components/submit-preprocess-job.jsonnet

```diff
@@ -1,75 +1,14 @@
-// Submit a Dataflow job to preprocess the input.
+// Submit a Dataflow job to preprocess the github public dataset
 local k = import "k.libsonnet";
 
+local experiments = import "experiments.libsonnet";
+local lib = import "submit-preprocess-job.libsonnet";
```

Comment:
What is the purpose of creating a submit-preprocess-job.libsonnet file?

Reply:
it's not necessary anymore. revert.